tests/e2e/lib/diskutils

```diff
@@ -106,7 +106,9 @@ select_disk_to_partition() {
   # Once a disk is found and partitioned, we need to notify the kernel about the changes
   if [[ -n "${DISK}" ]]; then
     echo "Rescanning partition table on /dev/${DISK}..."
-
+    if ! rpm -qf qm &>/dev/null; then
```

Comment:
@Yarboa in theory this is not required, this should be covered by the spec file "Requires: parted".